ci-cd-starter/vars/ci-cd-starter-vars.json

```diff
@@ -30,6 +30,16 @@
                 }
               },
               {
+                "filename": "https://raw.githubusercontent.com/rht-labs/openshift-templates/master/sonarqube-template.json",
+                "parameters": {
+                  "VOLUME_CAPACITY": "10Gi",
+                  "MEMORY_LIMIT": "2Gi",
```

Comment:
It's unclear the difference to me between `SONARQUBE_MEMORY_LIMIT` & `MEMORY_LIMIT` from reading this. That's a smell we need to update the template for readability. This is an FYI